### ISSUE TITLE
Table 1 Results on MR→ CT: (DSC and ASSD score)

### ISSUE THREAD

**Issue by nimradilawar** [aoge1993/SFDA-CCRC#1]
I reproduced the experiment exactly as described in your code.
Due to computational limitations, I am using a batch size of 14; all other settings are same as given. 
However, I am unable to reproduce the same results, or even results close to those reported in the paper Avg_DSC = 79.2 and Avg_ASSD = 5.8
I am using "train_targetWhs.py" to adapt the source model. I am specifically repeating the Table 1 experiment for MR → CT and using the same source model provided in the directory ./logs_train/mr/checkpoint_80.pth.tar.

Could you please help me understand what I might be missing or what additional steps I should take? When evaluating the teacher model, I am obtaining Avg_DSC = 63.93 and Avg_ASSD = 6.1, which are significantly different from the reported results.
I look forward to your response. Thank you

**Comment by zqp1226358** [comment]
@nimradilawar Hi Nimra, I think there may be two possible reasons.

1. Dataset-related issue: I’d recommend loading the dataset strictly following the format described in the README. Also, please double-check the orientation of the test set. In SIFA’s evaluation code (around line 159), it mentions ensuring that the orientation of the test data matches the training data. You can refer to: https://github.com/cchen-cc/SIFA/blob/master/evaluate.py

2. Environment / GPU differences could also affect the results. On my side, I’m using PyTorch 1.10.0.
I also reproduced the experiments last night, and my results were fairly close to those reported in the paper. Hopefully the suggestions above are helpful.

<img width="2409" height="831" alt="Image" src="https://github.com/user-attachments/assets/ec34951b-b18f-42d6-be25-41a029cefd7b" />
